tests/parm/input.mom6.nml.IN

```diff
@@ -11,5 +11,9 @@
          restart_output_dir = 'RESTART/',
          parameter_filename = 'INPUT/MOM_input',
                               'INPUT/MOM_override' /
+&nam_stochy
```

Comment:
This file is used by ufs-weather when run with the datm. Is the addition of nam_stochy and sfcperts required here? 

The coupled model uses either input.mom6_ccpp.nml.IN, input.benchmark_ccpp.nml.IN or input.benchmark_v16.nml.IN.

Reply:
Not in the PR, but will be when the ocean stochastic physics is merged in.